APITaxi2/__init__.py

```diff
@@ -169,7 +169,7 @@ def create_app():
             integrations=[
                 RedisIntegration(),
             ],
-            traces_sample_rate=1.0
+            traces_sample_rate=0.1
```

Comment:
Hoping it's not too much too fast, but it will be easy to adjust.

Reply:
We really send too much data to sentry right now, but yes we can increase later if we don't have enough data to analyze performances